packages/landing/src/landing.scss

```diff
@@ -388,17 +392,18 @@ header {
   }
 
   .pt-features {
-    padding: 0 25%;
+    .pt-feature-column {
+      width: calc(50% - #{$pt-grid-size * 2});
 
-    .pt-feature-column:nth-child(1),
-    .pt-feature-column:nth-child(2) {
-      float: none;
-      width: auto;
-      text-align: center;
-    }
+      &:nth-child(1) {
+        float: left;
+        margin-right: $pt-grid-size * 2;
+      }
 
-    .pt-feature-column:nth-child(2) {
-      padding-top: $pt-grid-size * 6;
+      &:nth-child(2) {
+        float: right;
```

Comment:
this looks exactly the same as lines 148-157 above. probably don't need to redefine the floats.